hedera-node/hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/pricing/BaseOperationUsage.java

```diff
@@ -381,8 +381,9 @@ UsageAccumulator cryptoUpdate(final int newAutoAssocSlots) {
                 .build();
         final var cryptoUpdateMeta = new CryptoUpdateMeta(canonicalTxn.getCryptoUpdateAccount(), now);
         final var into = new UsageAccumulator();
+        // Purposely don't include an explicit auto-assoc slot lifetime here
         CRYPTO_OPS_USAGE.cryptoUpdateUsage(
-                SINGLE_SIG_USAGE, NO_MEMO_AND_NO_EXPLICIT_XFERS, cryptoUpdateMeta, ctx, into);
+                SINGLE_SIG_USAGE, NO_MEMO_AND_NO_EXPLICIT_XFERS, cryptoUpdateMeta, ctx, into, 0);
```

Comment:
SHouldn't the base lifetime be 1 month ?

Reply:
The `0` here will leave the base usage computed the same.